### DISCUSSION TITLE
Guess 🇺🇸 state

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1073] [Galaxy Trivia] [answered]
🌋🌈🍍
🐝⛪⛷️
🚗🏭✋

**Comment by peterbaccaro** [comment]
1. **Hawaii**
2. **Utah**
3. **Michigan**

**Comment by thuongtruong1009** [comment, marked as answer]
🌋🌈🍍 - Hawaii is known for its volcanoes, frequent rainbows, and pineapples.
🐝⛪⛷️ - There might not be an exact match for the bee, church, and skier emojis, but Hawaii has a diverse range of activities and attractions.
🚗🏭✋ - The car, factory, and hand emojis might not have a direct connection to Hawaii, but transportation and industry do exist on the islands. The hand emoji could represent the "Aloha" spirit, which is significant in Hawaiian culture.

So, the best fit for the given emojis is Hawaii.
